### PR TITLE
PR for peer reviews

### PR DESCRIPTION
**Pull request by lederp23**

### CODE REVIEW

File: v1/accounts/views.py [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+
+def login(request):
+    """Logs a user in"""
+    authorized = False
+    username = request.form['username']
+    password = request.form['password']
+    if username is '' or password is '':
+        abort(400)
+    if not re.match('^[a-zA-Z0-9-_]*$', username):
```
Comment (RyanSept) [inline v1/accounts/views.py]: This ain't g@ng$t@

File: v1/api/models.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+class BucketList(db.Model):
+    """Model for bucketlists"""
+    __tablename__ = 'bucketlists'
+    __table_args__ = {'extend_existing': True}
+    id = db.Column(db.Integer, primary_key=True)
+    user_id = db.Column(db.Integer, db.ForeignKey('users.id'))
+    name = db.Column(db.String)
+    date_created = db.Column(db.DateTime, default=datetime.datetime.utcnow)
+    date_modified = db.Column(db.DateTime, default=datetime.datetime.utcnow)
+    created_by = db.Column(db.Integer, nullable=True)
```
Comment (RyanSept) [inline v1/api/models.py]: Why can this field be none? Is it possible to have bucketlists created by no one?

Comment (lederp23) [inline v1/api/models.py]: I was supposed to change that and I forgot. I will change it.

File: tests/test.py [hunk 3]
```diff
@@ -0,0 +1,396 @@
...
+from unittest import TestCase
+from flask_sqlalchemy import SQLAlchemy
+from app import app, db
+from v1.accounts.models import User
+from v1.api.models import BucketList, Item
+from urls import urls
+import json
+
+
+class MyTest(TestCase):
```
Comment (andela-bmwenda) [inline tests/test.py]: The Test class is quite long. Maybe you could have modularised the tests just as you have done with the models and views?

**Pull request merged**